### PR TITLE
tp-qemu: qemu_disk_img: disable those not supported versions for preallocation mode "full" and "falloc".

### PR DESCRIPTION
**Pull request by CongSmile**
Signed-off-by: Cong Li <coli@redhat.com>

id: 1285602

### CODE REVIEW

**Comment by CongSmile** [comment]
Hi @xiagao 

Could you help review this PR?

Thanks.

**Comment by xiagao** [comment]
Acked-by: <xiagao@redhat.com>

File: qemu/tests/cfg/qemu_disk_img.cfg [hunk 1]
```diff
@@ -37,8 +37,18 @@
                                 - metadata:
                                     preallocated = metadata
                                 - falloc:
+                                    no Host_RHEL.m5 Host_RHEL.m6.u6
+                                    no Host_RHEL.m6.u0 Host_RHEL.m6.u1
```
Comment (suqinhuang) [inline qemu/tests/cfg/qemu_disk_img.cfg]: Hi coli,

How about Host_RHEL.m6 directly?

Comment (CongSmile) [inline qemu/tests/cfg/qemu_disk_img.cfg]: Hi @suqinhuang 

According to @xiagao (feature owner), RHEL.6.7 and RHEL.6.8 and future versions need support this preallocation mode.

Thanks.

**Pull request merged**